readme.md

```diff
@@ -1,4 +1,28 @@
 <!-- express sẽ được sử dụng để tạo máy chủ
 nodemon sẽ giúp mình theo dõi các thay đổi đối với ứng dụng của mình bằng cách xem các tệp đã thay đổi và tự động khởi động lại máy chủ.
 dotenv để mình thêm các config cho database(host, port, user, pass, ...) và các config khác
-mysql để thao tác với database -->
\ No newline at end of file
+mysql để thao tác với database -->
+
+# Project Title
+Resful API with NodeJS, Express and MySql
+# Installing
+## Clone
+- Clone this repo to your local machine using `git clone https://github.com/hongquanhak/testAPI`
+### Set-up
+> Install npm or yarn packages
+```
+$ npm install
+$ yarn install
+```
+# Running the tests
+```
+$ npm start
+$ yarn start
+```
+# Including APIs
+- Endpoint: Get the rates with latest date.
```

Comment:
Em thêm enpoint cụ thể như Endpoint /api/v1/rates/latest: Get the rates with latest date